.github/workflows/build_and_test.yml

```
@@ -549,6 +549,7 @@ jobs:
         #   See also https://issues.apache.org/jira/browse/SPARK-38279.
         python3.9 -m pip install 'sphinx<3.1.0' mkdocs pydata_sphinx_theme ipython nbsphinx numpydoc 'jinja2<3.0.0' 'markupsafe==2.0.1'
         python3.9 -m pip install ipython_genutils # See SPARK-38517
+        python3.9 -m pip install 'docutils<1.7.0' # See SPARK-39421
```

successful: https://github.com/cxzl25/spark/runs/6794665905?check_suite_focus=true#step:11:466

failed: https://github.com/HyukjinKwon/spark/runs/6805386169?check_suite_focus=true#step:11:457

If we make sure it's `doctuils` problem, this might be 
```suggestion
        python3.9 -m pip install 'docutils<=0.17.1' # See SPARK-39421
```

---

See also https://pypi.org/project/docutils/#history

---

Thanks man